content/tbd/STRP.md

```diff
@@ -0,0 +1,38 @@
+---
+created: 2023-12-07
+contributors: Matthew McCarthy
+title: STRP
+url: https://strp.nl/
+locations:
+  - Eindhoven, NL
+start: "2006"
+end: 
+size: " 4-10"
+image: 
+sectors: Arts and Culture
+activities: Events and Retreats
+cohere: true
+metadata_version: "1"
+---
+
+# About
+
+STRP explores the relationship between art, technology, and major societal issues with the general public, artists, designers, media makers, and thinkers. The Eindhoven-based organisation starts dialogue about speculative future scenarios with various events throughout the year and an annual festival in the spring.
+
+![[assets/strp.jpg]]
+
+# Theory of Change
+
+STRP believes that systems change begins with dialogue, artistic expression. and building community. Since 2006, STRP has been offering new scenarios for the future, dialogue sessions, a festival, an international exhibition, workshops, tours, an extensive educational program, music and performances. Every year they co-produce several new works of art in collaboration with emerging artists.
+
+# Key Educational Resources
```

Comment:
could you pick out one or two topics or themes that their educational offerings address at least? so i get a sense of their educational offers and what kinds of subjects they deal with ?